demo_nodes_cpp/README.md

```diff
@@ -37,6 +37,13 @@ Run the command below to compile the `demo_nodes_cpp` ROS 2 package:
 colcon build --packages-up-to demo_nodes_cpp
 ```
 
+**Note**: By default, the demo executables will spin up the [SingleThreaded executor](https://docs.ros.org/en/rolling/Concepts/Intermediate/About-Executors.html#executors) if run in separate processes, i.e., not composed in the same component container.
+To configure the demo executables to run with a different executor, build the package with the custom `DEMO_EXECUTOR` flag set to the fully qualified name of the executor.
+For example, to run with the experimental `EventsExecutor`,
+```bash
+colcon build --packages-up-to demo_nodes_cpp --cmake-args -DDEMO_EXECUTOR:STRING=rclcpp::experimental::executors::EventsExecutor
+```
```

Comment:
can we just rebuild this package?

```suggestion
For example, to run with the experimental `EventsExecutor`,

```bash
colcon build --packages-select demo_nodes_cpp --cmake-args -DDEMO_EXECUTOR:STRING=rclcpp::experimental::executors::EventsExecutor
```
```